.gitignore

```diff
@@ -8,3 +8,5 @@ node_modules
 !.env.example
 vite.config.js.timestamp-*
 vite.config.ts.timestamp-*
+/localstack/local-data
+/prisma/migrations/
```

Comment:
I saw in the Git history of this project that previously prisma migrations were committed to the repo. Not sure what's the current process is now, so I decided put in the Git Ignore for now. Let me know what you think.